spec/lib/vcr_spec.rb

```diff
@@ -341,9 +341,9 @@ def insert_cassette(name = :cassette_test)
   end
 
   describe '.turned_on?' do
-    it 'is on by default' do
+    it 'is off by default' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.